src/firebase.js

```diff
@@ -14,5 +14,6 @@ const firebaseApp = firebase.initializeApp({
 const db = firebaseApp.firestore();
 const auth = firebase.auth();
 const storage = firebase.storage();
+const provider = new firebase.auth.GoogleAuthProvider();
 
-export { db, auth, storage };
+export { db, auth, storage,provider };
```

Comment:
Name all the providers with their name, So export this like,
```jsx
const GoogleAuthProvider = new firebase.auth.GoogleAuthProvider();

export { db, auth, storage, GoogleAuthProvider };
```

Do this like so since I am adding Twitter and Facebook Auth.